test/e2e_node/jenkins/jenkins-docker-validation.properties

```diff
@@ -1,15 +1,19 @@
 GCI_IMAGE_PROJECT=container-vm-image-staging
-GCI_IMAGE_FAMILY=gci-preview-test
+GCI_IMAGE_FAMILY=gci-canary-test
 GCI_IMAGE=$(gcloud compute images describe-from-family ${GCI_IMAGE_FAMILY} --project=${GCI_IMAGE_PROJECT} --format="value(name)")
 DOCKER_VERSION=$(curl -fsSL --retry 3 https://api.github.com/repos/docker/docker/releases | tac | tac | grep -m 1 "\"tag_name\"\:" | grep -Eo "[0-9\.rc-]+")
 GCI_CLOUD_INIT=test/e2e_node/jenkins/gci-init.yaml
+GCE_IMAGE_CONFIG="images:
+  gci-canary:
```

Comment:
Not sure what this config file is used for, but why `gci-canary` not `gci-canary-test`? The image family `gci-canary` is used for GCI base images while `gci-canary-test` for test images.